lint/lint_test.go

```diff
@@ -103,11 +99,7 @@ func TestLinter_run(t *testing.T) {
 }
 
 func TestLinter_LintProtos_RulePanics(t *testing.T) {
-	fd, err := builder.NewFile("test.proto").Build()
-	if err != nil {
-		t.Fatalf("Failed to build the file descriptor.")
-	}
-
```

Comment:
@quirogas  This should actually be using the descriptor builders not this template-based `buildFile` function.

https://go.dev/play/p/GFheUaTmFuE